### PR TITLE
cleans up code and adds Martin Grotzke to the list of Authors

### PR DESCRIPTION
**Pull request by timoe**
## Description
Adds `final` modifiers, removes empty lines and extends the list of autors.

---

@agudian: Please review, merge and delete branch.

### CODE REVIEW

File: src/main/java/com/adobe/testing/s3mock/FileStoreController.java [hunk 1]
```diff
@@ -101,7 +101,8 @@
    */
   @RequestMapping(value = "/", method = RequestMethod.GET, produces = {
       "application/x-www-form-urlencoded"})
-  public @ResponseBody ListAllMyBucketsResult listBuckets() {
+  public @ResponseBody
+  ListAllMyBucketsResult listBuckets() {
```
Comment (agudian) [inline src/main/java/com/adobe/testing/s3mock/FileStoreController.java]: line break here?

File: src/main/java/com/adobe/testing/s3mock/FileStoreController.java [hunk 2]
```diff
@@ -335,7 +337,8 @@ private boolean isV4SigningEnabled(final HttpServletRequest request) {
           NOT_SERVER_SIDE_ENCRYPTION
       },
       produces = "application/x-www-form-urlencoded; charset=utf-8")
-  public @ResponseBody CopyObjectResult copyObject(@PathVariable final String destinationBucket,
+  public @ResponseBody
+  CopyObjectResult copyObject(@PathVariable final String destinationBucket,
```
Comment (agudian) [inline src/main/java/com/adobe/testing/s3mock/FileStoreController.java]: line break?

File: src/main/java/com/adobe/testing/s3mock/FileStoreController.java [hunk 3]
```diff
@@ -369,7 +372,8 @@ private boolean isV4SigningEnabled(final HttpServletRequest request) {
           SERVER_SIDE_ENCRYPTION
       },
       produces = "application/x-www-form-urlencoded; charset=utf-8")
-  public @ResponseBody CopyObjectResult copyObject(
+  public @ResponseBody
+  CopyObjectResult copyObject(
```
Comment (agudian) [inline src/main/java/com/adobe/testing/s3mock/FileStoreController.java]: line break?

**Comment by timoe** [comment]
updated

File: src/main/resources/application.properties [hunk 4]
```diff
@@ -13,10 +13,8 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 #
-
```
Comment (agudian) [inline src/main/resources/application.properties]: For some reason, this seems to break the license check. please add the newline here again.

**Pull request merged**